lib/ActivityStreams.js

```diff
@@ -443,6 +470,19 @@ ActivityStreams.prototype = {
   },
 
   /**
+    * Start a timer to fetch a new recommendation every hour. This will only
+    * run for those in the experiment
+    */
+  _startRecommendationRefresh(recommendationTTL) {
+    if (recommendationTTL && simplePrefs.prefs.recommendations) {
```

Comment:
in that case, then if the preff is off at the start, the timeout would never start, and that function wouldn't be called again and you would never refresh recommendations
right?